docs/getting-started-9c-with-the-unity-editor/getting-started-9c-with-the-unity-editor.md

```diff
@@ -0,0 +1,256 @@
+
+  <img
+    alt="Fork: `Remotes > Origin > main` -> `Checkout 'origin/main'...` menu"
+    src="./images/fork-checkout-main.png"
+    style="width:50vw" />
+
+- In the following popup, click the `Track` button.
+
+  <img
+    alt="Fork: `Track` popup"
+    src="./images/fork-track-popup.png"
+    style="width:50vw" />
+
+# Play the Nine Chronicles with the Unity editor
+
+## Open the Nine Chronicles project with the Unity Hub
+
+### 1. Run the Unity Hub.
+
+Unity account creation and licensing are not covered here. You can also proceed for free(Unity Personal License).
+
+### 2. Open the Nine Chronicles project.
+
+- Click the `Projects > Open` button when running for the first time.
+
+  <img
+    alt="Unity Hub: `Projects > Open` button"
+    src="./images/unity-hub-projects-open.png"
+    style="width:50vw" />
+
+- Select the `/nekoyume` folder in the location where you cloned the Nine Chronicles repository and click the `Open` button.
+
+  <img
+    alt="Unity Hub: `Open Project` popup"
+    src="./images/unity-hub-projects-open-popup.png"
+    style="width:50vw" />
+
+## Configure `clo.json`
+
+The clo.json file is a file that sets various options related to the blockchain network.
+
+### 1. Create a clo.json file in `/nekoyume/Assets/StreamingAssets/` based on the cloned path.
+
+### 2. Fill in the clo.json file as shown below and save it.
+
+```json
+{
+    "GenesisBlockPath": "https://download.nine-chronicles.com/genesis-block-Nine Chronicles-main",
+    "NoMiner": true,
+    "RpcClient": true,
+    "RpcServerHost": "9c-main-rpc-1.nine-chronicles.com",
+    "RpcServerPort": 31238,
+    "ApiServerHost": "https://api.9c.gg/graphql"
+}
+```
+
+## Prepare the key in the `Key Store` path
+
+In order to play the Nine Chronicles, you need a Private Key to participate in the blockchain network.
+If you've already played the Nine Chronicles, that means you already have a Private Key and a Protected Private Key.
+
+- Private Key: A key required to sign a transaction.
+- Protected Private Key: This is the encrypted file of the private key.
+
+### 1. Generate Private Key
+
+- Skip this step and go to the [Play](#play) step to use the step to generate a private key.
+
+### 2. Put your Protected Private Key into the Key Store path
+
+- Key Store path
+  - The path to keep the Protected Private Key.
+  - The default value of this path is determined [this way] [default-path-of-key-store] based on [Libplanet 0.41.0 version][libplanet-0.41.0].
+  - Windows
+    - `%appdata%\planetarium\keystore\`
+  - Mac
+    - `/Users/USERNAME/.config/planetarium/keystore/` or
+    - `/Users/USERNAME/Library/Application Support/planetarium/keystore/`
+  - Linux
+    - `home/planetarium/keystore/` or
+    - `home/.config/planetarium/keystore/`
+
+[default-path-of-key-store]: https://github.com/planetarium/libplanet/blob/0.41.0/Libplanet/KeyStore/Web3KeyStore.cs#L21-L29
+[libplanet-0.41.0]: https://github.com/planetarium/libplanet/tree/0.41.0
```

Comment:
Do we have to specify version? It's to hard to edit version everytime libplanet has bumped.

Reply:
@U-lis The logic to determine the Key Store path is in libplanet. I wrote the description based on the most recent version, and if this logic changes later, I thought to change the libplanet version and description together.
If we link the `main` branch on behalf of a specific version, there might be differences in the description and the logic the link points to, and I'd rather avoid this.